src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1641,7 +1653,7 @@ public void TestPeakVirtualMemorySize()
             CreateDefaultProcess();
 
 #pragma warning disable 0618
-            AssertNonZeroWindowsZeroUnix(_process.PeakVirtualMemorySize);
+            AssertNonZeroAllZeroDarwin(_process.PeakVirtualMemorySize);
```

Comment:
Is there any reliable way to test that any of these is increasing when process does something specific?
If not, did we at least do some manual testing to verify the values look reasonable?

Reply:
Not sure about an automated way to do so. I did verify this manually by checking the values in `/proc/[pid]/status`.